webapp/static/interop-data.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
Any thoughts about splitting this out to a file per year?

It'll make loading it in interop.js a little weird though. But something I am thinking about.

Reply:
We can create a /interop-data/{year} endpoint if we want to split the file

Reply:
Is this something we'd like to do now, or is it better suited for a separate change? I think it's probably a wise idea

Reply:
I am happy with a follow-up change on this

Reply:
we can follow up on it. could you create an issue about it?

Reply:
[Issue created](https://github.com/web-platform-tests/wpt.fyi/issues/2974)

Reply:
[Issue created](https://github.com/web-platform-tests/wpt.fyi/issues/2974)